components/doc/autocomplete/templatedoc.js

```diff
@@ -35,14 +35,18 @@ export function TemplateDoc(props) {
         );
     };
 
+    const panelFooterTemplate = () => {
+        return <div className="py-2 px-3">Footer Template'</div>;
```

Comment:
Can you make it similar to the Dropdown example it make the use case more realistic.

Reply:
Unlike Dropowed, In AutoCompleted `onChange` event fires whenever we type. So `selectedCountry` would've value `a` if I typed a. 

What should be the text here `a entered`, `a typed` or do you have any suggestions?

Reply:
I can also do this 

```
   const panelFooterTemplate = () => {
        const isCountrySelected = (filteredCountries || []).some( country => country['name'] === selectedCountry );
           return (
            <div className="py-2 px-3">
                {isCountrySelected ? (
                    <span>
                        <b>{selectedCountry}</b> selected.
                    </span>
                ) : (
                    'No country selected.'
                )}
            </div>
        );
    };
 ```

Reply:
Yep that is what I was thinking.